### PR TITLE
Security Upgrade

### LINKED ISSUE #4281: Security vulnerability on prebuild-install dependency.
### How frequently does the bug occur?

All the time

### Description

Hi, 

You need to consider using a later version of prebuild-install as the  "^6.1.1" that you are currently using, has a "High Severity" dependency vulnerability that comes from the simple-get < 4.0.1 version. 

Here is the respective CVE-2022-0355 https://github.com/advisories/GHSA-wpg7-2c88-r8xv

### Stacktrace & log output

_No response_

### Can you reproduce the bug?

Yes, always

### Reproduction Steps

_No response_

### Version

N/A

### What SDK flavour are you using?

MongoDB Realm (i.e. Sync, auth, functions)

### Are you using encryption?

Yes, using encryption

### Platform OS and version(s)

N/A

### Build environment

Which debugger for React Native: ..

### Cocoapods version

N/A

### PR DESCRIPTION
**Pull request by takameyer**
## What, How & Why?
`prebuild-install` < 7 was using an insecure `simple-get` package.
Upgrade also potentially fixes some `electron` issues.
Is not compatible with node versions 10 and under.
https://github.com/prebuild/prebuild-install/releases

This closes #4281

## ☑️ ToDos

* [x] 📝 Changelog entry
* [ ] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 🚦 Tests
* [ ] 📱 Check the React Native/other sample apps work if necessary
* [ ] 📝 Public documentation PR created or is not necessary
* [ ] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [ ] typescript definitions file is updated
* [ ] jsdoc files updated
* [ ] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,67 +1,78 @@
-x.x.x Release notes (yyyy-MM-dd)
```
Comment (kraenhansen) [inline CHANGELOG.md]: I suggest ignoring this file in your markdown linter 🙂

Comment (takameyer) [reply CHANGELOG.md]: Oh no... 🙈

**Review by kneth — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kneth) [reply CHANGELOG.md]: My :+1: is based on Kræn's suggestion above 😄

Comment (takameyer) [reply CHANGELOG.md]: Done

**Comment by ericanderson** [comment]
Can we ship it?

**Pull request merged**